packages/runtime/container-runtime/src/summaryGenerator.ts

```diff
@@ -349,7 +349,7 @@ export class SummaryGenerator {
                 // back-compat: cast needed until dep on protocol-definitions version bump
```

Comment:
can you check the history for this comment for how long ago it was?  the code below on L351 is handling the back-compat stuff already so if this is really safe to remove this whole block should be cleaned up